### ISSUE TITLE
groupings

### ISSUE THREAD

**Issue by earloc** [earloc/TypealizR#14]
As a developer, I want to specify a `group-key` that the source-generator uses to group generated extension-methods together, in order to let me better organize larger resx-files.

**Comment by earloc** [comment]
There could be an attribute-style notation which could be put in front of a parameter-expression, like 
-`[Message] Greet {userName}`
-`[Info] {userName} has been signed out`
-`[Warning] {userName} will be signed out`


suche strings would then be callable via:

```csharp
IStringlocalizer<App> localize;

localize. Message() . Greet("Arthur") ;
localize. Info(). userName__has_been_signed_out("Arthur") ;
localize. Warning(). userName_will_be_signed_out("Arthur") ;
```

**Comment by earloc** [comment]
for later finding how to use nugets in the generator:
https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.cookbook.md#use-functionality-from-nuget-packages

**Comment by earloc** [comment]
Closed by #64
